src/EnergyPlus/SingleDuct.cc

```diff
@@ -1791,8 +1791,11 @@ namespace SingleDuct {
 		//Do a check and make sure that the max and min available(control) flow is
 		//  between the physical max and min while operating.
 		SysInlet( SysNum ).AirMassFlowRateMaxAvail = min( Sys( SysNum ).AirMassFlowRateMax, Node( InletNode ).MassFlowRateMaxAvail );
-		SysInlet( SysNum ).AirMassFlowRateMinAvail = min( max( Node( OutletNode ).MassFlowRateMin, Node( InletNode ).MassFlowRateMinAvail ), SysInlet( SysNum ).AirMassFlowRateMaxAvail );
-
+		if ( Sys( SysNum ).SysType_Num == SingleDuctConstVolReheat ) { // AirTerminal:SingleDuct:ConstantVolume:Reheat
```

Comment:
@lgu1234 @rraustad Well, I learned something new.  I never realized that adding the OA unit without an exhaust fan (either in the unit or in the zone) could impact the air loop by creating extra return flow.  That shouldn't be able to drive the air loop to higher flow - but that's (kind of) a different issue.  I would suggest for this particular fix, that the OA units could check if there is no exhaust fan in the OA unit, and there is no zone exhaust fan, then issue a warning.  It might be what the user wants.  In this particular case, it is not what the user (moi) wanted.  And said user was very confused and frustrated by the odd result.  But I think I also have a case with ideal loads that shows similar symptoms.  I'll add that defect file shortly.